src/NuGet.Core/NuGet.Protocol/Model/LocalPackageSearchMetadata.cs

```diff
@@ -42,6 +42,8 @@ public LocalPackageSearchMetadata(LocalPackageInfo package)
 
         public Uri LicenseUrl => Convert(_nuspec.GetLicenseUrl());
 
+        public List<string> OwnersList => null;
```

Comment:
This can be computed from `Owners`. While we might not use it in our own code, from an API design point of view, it would look like a bug if this returns null (or even an empty list) when `Owners` returns a non-empty value.

Reply:
Am I misunderstanding Local metadata? I believe it's always metadata from a `nuspec` on the client machine; therefore, it should never have OwnersList, since OwnersList is a package source concept (Search API).

Reply:
There was a time when owners wasn't deprecated from the nuspec. Some customers might be using it for reasons other than what nuget.org uses it for. But if we want to maintain the line that owners is now deprecated, then perhaps this method should throw a `NotSupportedException`, and the `Owners` property should also be marked as obsolete, so customers using NuGet.Protocol in their own apps are aware.

Reply:
You're talking about `Owners` but you're pointing at `OwnersList`. I think `NotSupportedException` is reasonable, as local metadata will never support a server's metadata. 
 
Marking `Owners` obsolete seems like a separate concern than what this PR does, which is _adds_ server metadata. We can discuss if we're ready to officially make the nuspec field obsolete. Regardless, that will be a separate Spec and separate PR to this one.

Reply:
I've updated `LocalPackageSearchMetadata` to throw when accessing the list.

I'd like to update `PackageSearchMetadataV2Feed` as well, but there's a design problem that needs addressed.
The utility for V2 feeds here:
https://github.com/NuGet/NuGet.Client/blob/4b618ca3c42fa0b678b28d17af2d815c1d26d28b/src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedUtilities.cs#L25

...causes the Cloned metadata to copy properties here (which is fine for V3 feeds, but not for V2 feeds):
https://github.com/NuGet/NuGet.Client/blob/4b618ca3c42fa0b678b28d17af2d815c1d26d28b/src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataBuilder.cs#L95

Since I don't want this code to throw `NotImplementedException`, a V2 feed will need to continue passing `null`.

Reply:
>  and they might not notice the bug during development

Are you seeing a bug, or think something may be a bug, or is this just hypothetical?

I've offered in a comment last week to meet with you and discuss this PR directly. I think it may be more efficient to do so. Could you reach out to me with a time that works?